src/Orleans/Telemetry/Consumers/ConsoleTelemetryConsumer.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Orleans.Runtime
+{
+    public class ConsoleTelemetryConsumer : ITraceTelemetryConsumer, IExceptionTelemetryConsumer
```

Comment:
Both classes write to a `TextWriter` as you can see on `System.Console` here http://referencesource.microsoft.com/#mscorlib/system/console.cs,f907d79481da6ba4 and the `TraceListener` http://referencesource.microsoft.com/#System/compmod/system/diagnostics/TextWriterTraceListener.cs,7fdde189840b2aa2 as well. The diff is that `Console` also accepts input (which we are not going to use anyway) and it has its own native locks as you can see on the reference source.

Running a few tests in a loop here I didn't saw a diff on the perf explorer chart. You will probably be able to check that on your perf tests.